test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeArrayTests.java

```diff
@@ -357,317 +357,317 @@ static public void testValueTypeArrayAssignments() throws Throwable {
 		}
 	}
 
-   static primitive class SomePrimitiveClassWithDoubleField{
-      public double d;
+	static primitive class SomePrimitiveClassWithDoubleField{
+		public double d;
 
 		SomePrimitiveClassWithDoubleField(double x) {
 			this.d = x;
 		}
-   }
+	}
 
-   static primitive class SomePrimitiveClassWithFloatField{
-      public float f;
+	static primitive class SomePrimitiveClassWithFloatField{
+		public float f;
 
 		SomePrimitiveClassWithFloatField(float x) {
 			this.f = x;
 		}
-   }
+	}
 
-   static primitive class SomePrimitiveClassWithLongField{
-      public long l;
+	static primitive class SomePrimitiveClassWithLongField{
+		public long l;
 
 		SomePrimitiveClassWithLongField(long x) {
 			this.l = x;
 		}
-   }
+	}
 
-   static class SomeIdentityClassWithDoubleField{
-      public double d;
+	static class SomeIdentityClassWithDoubleField{
+		public double d;
 
 		SomeIdentityClassWithDoubleField(double x) {
 			this.d = x;
 		}
-   }
+	}
 
-   static class SomeIdentityClassWithFloatField{
-      public float f;
+	static class SomeIdentityClassWithFloatField{
+		public float f;
 
 		SomeIdentityClassWithFloatField(float x) {
 			this.f = x;
 		}
-   }
+	}
 
-   static class SomeIdentityClassWithLongField{
-      public long l;
+	static class SomeIdentityClassWithLongField{
+		public long l;
 
 		SomeIdentityClassWithLongField(long x) {
 			this.l = x;
 		}
-   }
+	}
 
    interface SomeInterface1WithSingleImplementer {}
 
    interface SomeInterface2WithSingleImplementer {}
```

Comment:
It looks like these two interface declarations are the last two lines that are still indented with spaces.